config/crd/bases/limitador.3scale.net_limitadors.yaml

```diff
@@ -0,0 +1,58 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: limitadors.limitador.3scale.net
+spec:
+  group: limitador.3scale.net
+  names:
+    kind: Limitador
+    listKind: LimitadorList
+    plural: limitadors
+    singular: limitador
+  scope: Namespaced
```

Comment:
I understand then you want to be able to have the ability to deploy multiple Limitadors in different namespaces

Reply:
I included this point in my original comment because I don't really have an answer for this yet. This is something we'll need to analyze and see whether it makes sense or if it makes things too complicated.

Reply:
:+1: